src/js/helpers/d3bar.js

```diff
@@ -137,7 +137,14 @@ class BarChart extends Chart {
       .attr('x', d => x(this.label(d)))
       .attr('y', chartHeight)
       .attr('width', x.bandwidth())
-      .attr('height', 0);
+      .attr('height', 0)
+      .attr('class', d => {
+        const className = this.label(d)
+          .toLowerCase()
+          .replace(/[^\w]+/g, '_')
```

Comment:
This will replace all whitespace with `_` so that the replace below will not do anything. eg: 

```javascript
> 'a space'.replace(/[^\w]+/g, '_').replace(/\s+/g, '-')
'a_space'
```

Perhaps I'm missing some other case though :)